ts/src/local/conductor.ts

```diff
@@ -249,14 +249,12 @@ export class Conductor implements IConductor {
     );
     const startPromise = new Promise<void>((resolve) => {
       runConductorProcess.stdout.on("data", (data: Buffer) => {
-        logger.debug(`starting conductor\n${data}`);
-
         const numberMatches = data
           .toString()
           .match(/Running conductor on admin port (\d+)/);
         if (numberMatches) {
           this.adminApiUrl.port = numberMatches[1];
-          logger.debug(`admin port ${this.adminApiUrl.port}\n`);
+          logger.debug(`starting conductor\n${data}`);
```

Comment:
wouldn't you still want the above line that you removed, perhaps?

Reply:
The hc sandbox command outputs the admin port anyway:
```bash
16:45:20 [Tryorama - Local Conductor] debug: starting conductor

Conductor ready.
hc-sandbox: Running conductor on admin port 60042
```